pkg/api/jobservice/jobservice.proto

```diff
@@ -0,0 +1,44 @@
+syntax = 'proto3';
+
+package jobservice;
+
+message JobServiceRequest {
+    string job_id = 1;
+    string job_set_id = 2;
+    string queue = 3;
+}
+
+message JobServiceResponse {
+// See event.proto for a list of possible events
+// If you add new events to events.proto please add them here also.
+    enum State {
```

Comment:
What about UNABLE_TO_SCHEDULE?  Does that translate to a Failed?  

I guess I took a Failed job to be a job that runs and returns a failed error code.  and not being able to schedule a job due to a bad Pod would be UNABLE_TO_SCHEDULE?

Reply:
`UNABLE_TO_SCHEDULE`  means that Armada couldn't schedule the job on a cluster.  It may retry the job (in which case you'll get Failed, Cancelled, Succeeded depending on what happens to the retry) or it may be that the job has already been retried enough times so you'll get Failed.